bundles/edu.kit.ipd.sdq.activextendannotations/src/edu/kit/ipd/sdq/activextendannotations/DelegateProcessorUtil.xtend

```diff
@@ -1,65 +0,0 @@
-package edu.kit.ipd.sdq.activextendannotations
-
-import org.eclipse.xtend.lib.macro.TransformationContext
-import org.eclipse.xtend.lib.macro.declaration.MutableClassDeclaration
-import org.eclipse.xtend.lib.macro.declaration.ResolvedMethod
-import org.eclipse.xtend.lib.macro.declaration.TypeReference
-import java.util.Map
-
-// TODO MK if active annotation for static delegation is added to Xtend then
-// merge all code of this class with org.eclipse.xtend.lib.annotations.DelegateProcessor$Util
```

Comment:
please add similar comments to your code to make clear which code could be merged with code for @Delegate if this active annotation became part of Xtend